rfcs/text/0001-wildcard-data-type.md

```diff
@@ -150,42 +152,26 @@ The following table is a comparison of `wildcard` vs. `keyword` [2]:
 | Supports normalizers in mappings | Y | N |
 | Indexing speeds | Fast | Slower (see *6) |
 
-1. Somewhat slower as doc values retrieved from compressed blocks of 32
-2. Somewhat slower because approximate matches with n-grams need verification
-3. Keyword field visits every unique value only once but wildcard field assesses every utterance of values
-4. If "allow expensive queries" is enabled
-5. Depends on common prefixes - keyword fields have common-prefix based compression whereas wildcard fields are whole-value LZ4 compression.
-6. Will vary with content but a test indexing weblogs took 499 seconds vs. keyword's 365 seconds.
+1. Somewhat slower as doc values retrieved from compressed blocks of 32.
+2. Somewhat slower because approximate matches with n-grams need verification.
+3. Keyword field visits every unique value only once, but wildcard field assesses every utterance of values.
```

Comment:
what's an utterance?

Reply:
The table and its footnotes are taken from the [wildcard intro blog post](https://www.elastic.co/blog/find-strings-within-strings-faster-with-the-new-elasticsearch-wildcard-field), and I believe the point here is that for a wildcard field, Elasticsearch will need to traverse all the 3-character sequences indexed from the original string.